lib/db/schema/patch-067-068.sql

```diff
@@ -0,0 +1,165 @@
+ALTER TABLE `unverifiedTokens`
+ADD COLUMN `tokenVerificationCodeHash` BINARY(32) DEFAULT NULL,
+ADD COLUMN `tokenVerificationCodeExpiresAt` BIGINT UNSIGNED DEFAULT NULL,
+ALGORITHM = INPLACE, LOCK = NONE;
+
+CREATE INDEX `unverifiedTokens_tokenVerificationCodeHash_uid`
```

Comment:
Did you intend to make this a `CREATE UNIQUE INDEX`?  If not, I don't think it needs to contain the `uid` column at all.